### PR TITLE
Update tr.po

### PR DESCRIPTION
**Pull request by TufanSirin**
Hi Gunter I'm back :) 
I want to complete tr.po

### CODE REVIEW

**Comment by gunterkoenigsmann** [comment]
Will try to make the new translation build.

File: locales/tr.po [hunk 1]
```diff
@@ -4753,23 +4759,26 @@ msgstr ""
 msgid ""
 "The load() command of Maxima versions >5.38 is able to load .wxm files as "
 "library of maxima's commands."
-msgstr ""
+msgstr "5.38 den yukarı Maxima versiyonları .wxm dosyalarını"
+"maxima komutlarının kütüphanesi olarak yükler"
 
 #: /home/gunter/src/wxmaxima/src/TextCell.cpp:188
```
Comment (gunterkoenigsmann) [inline locales/tr.po]: Attention:
 +msgstr "5.38 den yukarı Maxima versiyonları .wxm dosyalarını"
		+"maxima komutlarının kütüphanesi olarak yükler"
		 
Doesn't introduce a space in front of Maxima which is a thing that has completely taken me by surprise when adding the new string.

**Comment by gunterkoenigsmann** [comment]
Welcome back ;-)
It is good to have translators. And I felt guilty for having changed and added this many strings this time.

In the meantime I have corrected the two places gettext would have caused missing spaces in in the translation

Kind regards,
Gunter.

**Comment by TufanSirin** [comment]
I will go on to do translations as much as I can :)

**Pull request merged**